exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
Should this gem adopt Bundler's gem naming conventions? http://guides.rubygems.org/name-your-gem/

It seems like the classes are named appropriately for underscores, and would just need to change directories a little bit.